test/replication/qsync_basic.result

```diff
@@ -270,7 +279,7 @@ test_run:switch('default')
  | ---
  | - true
  | ...
-box.cfg{replication_synchro_timeout = 0.001, replication_synchro_quorum = 3}
+box.cfg{replication_synchro_timeout = 0.1, replication_synchro_quorum = 3}
```

Comment:
Better leave a tiny timeout here. Quorum is 3 so there is no chance the transaction below gets committed. No need to wait for extra `0.1` second there

Reply:
Yep, the idea behind was the async followed after sync can be either in or not in the limbo. That's not true, fixed.